appserver/website-bootstrap/network.js

```diff
@@ -1,3 +1,218 @@
+                let i = 0;
+                for(i; i < retweets.valueOf(); i++){
+                    let node = {};
+                    let link = {};
+                    node.id = network[i].username;
+                    nodes.push(node);
+                    link.source = uname;
+                    link.target = network[i].username;
+                    link.value = network[i].retweet_count;
+                    links.push(link);
+                }
+                networkData.nodes = nodes;
+                networkData.links = links;
+                // alert(JSON.stringify(networkData)); // check that data was correctly parsed
+                graphNetwork(networkData)
+            });
+        }
+    });
+
+    const graphData = {
+        "nodes": [
+            {"id": "n1"},
+            {"id": "n2"},
+            {"id": "n3"},
+            {"id": "n4"},
+            {"id": "n5"},
+            {"id": "n6"},
+            {"id": "n7"},
+            {"id": "n8"},
+            {"id": "n9"},
+            {"id": "n10"},
+            {"id": "n11"},
+            {"id": "n12"},
+            {"id": "n13"},
+            {"id": "n14"},
+            {"id": "n15"},
+            {"id": "n16"},
+            {"id": "n17"},
+            {"id": "n18"},
+            {"id": "n19"},
+            {"id": "n20"}
+            ],
+        "links": [
+            {"source": "n1", "target": "n2", "value": 1},
+            {"source": "n1", "target": "n12", "value": 3},
+            {"source": "n1", "target": "n13", "value": 2},
+            {"source": "n1", "target": "n11", "value": 30},
+            {"source": "n13", "target": "n2", "value": 1},
+            {"source": "n15", "target": "n2", "value": 12},
+            {"source": "n4", "target": "n20", "value": 1},
+            {"source": "n17", "target": "n2", "value": 10},
+            {"source": "n18", "target": "n20", "value": 1},
+            {"source": "n19", "target": "n12", "value": 1},
+            {"source": "n19", "target": "n9", "value": 1},
+            {"source": "n11", "target": "n17", "value": 19},
+            {"source": "n9", "target": "n20", "value": 1},
+            {"source": "n17", "target": "n5", "value": 11},
+            {"source": "n1", "target": "n14", "value": 3},
+            {"source": "n1", "target": "n15", "value": 7},
+            {"source": "n1", "target": "n16", "value": 12},
+            {"source": "n2", "target": "n4", "value": 7},
+            {"source": "n3", "target": "n5", "value": 9},
+            {"source": "n2", "target": "n10", "value": 7},
+            {"source": "n3", "target": "n9", "value": 7},
+            {"source": "n4", "target": "n8", "value": 20},
+            {"source": "n4", "target": "n7", "value": 15},
+            {"source": "n8", "target": "n1", "value": 1},
+            {"source": "n6", "target": "n1", "value": 1}
+            ]
+    };
+    graphNetwork(graphData)
+});
+
+function graphNetwork(data) {
```

Comment:
just curious- why pass the nodes & links inside a single wrapper object rather than as two separate arguments?

Reply:
Note that this is not necessarily a request for change.

Reply:
uhh idk! Jen?